tests/dummy/app/templates/application.hbs

```diff
@@ -32,6 +32,7 @@
         <div class="install-wrapper">
           <code class="install-cmd">ember install ember-cli-clipboard</code>
           {{#copy-button
+            title='copy direct'
```

Comment:
No problem, sorry I didn't see that the docs were separate from the component itself. Should be fixed.